Druid/DRIRGenerator.class.st

```diff
@@ -1390,7 +1392,10 @@ DRIRGenerator >> interpretSumSmallIntegerWithOverflowWith: aRBMessageNode [
 { #category : #'special cases' }
 DRIRGenerator >> interpretTrueObjectWith: aRBVariableNode [
 
-	^ self addCopy: aRBVariableNode with: DRTrueObject new
+	^ self
+		  addCopy: aRBVariableNode
+		  with:
+		  (DRObjectReferenceValue expression: 'objectMemory trueObject')
```

Comment:
Starts looking good :)

Reply:
I didn't worry about the cleaning because you already changed this hierarchy 👍